packages/mobile/src/identity/reducer.ts

```diff
@@ -89,6 +93,8 @@ export interface State {
   e164NumberToAddress: E164NumberToAddressType
   e164NumberToSalt: E164NumberToSaltType
   addressToDataEncryptionKey: AddressToDataEncryptionKeyType
+  // Doesn't contain all known addresses, use only as a fallback.
+  addressToDisplayName: AddressToDisplayNameType
```

Comment:
Could you add a `TODO` note about removing this once we have CIP8 working in the app?